README.md

```diff
@@ -83,29 +93,18 @@ Operations Team takes care of legal documents, invoicing and remittances.
 
 1. **Application:**
    1. [Fork](https://github.com/w3f/Open-Grants-Program/fork) this repository
-   2. In the newly created fork, create a **copy** of the [application-template.md](./applications/application-template.md). In other words: In the GitHub web interface, you need to create a new file and copy the content of the [application-template.md](./applications/application-template.md) inside the new file. _Don't change the template file directly!_
-   3. Label the file as "project_name.md".
-   4. Fill out the template with the details of your project. The more information you provide, the faster the review.
+   2. In the newly created fork, create a **copy** of the application template ([`applications/application-template.md`](./applications/application-template.md)). If you're using the GitHub web interface, make sure you create a new file and copy the content of the template inside the new file. _Don't change the template file directly!_
```

Comment:
> If you're using the GitHub web interface

Doesn't that apply to anyone?

Reply:
Nice catch.

```suggestion
   2. In the newly created fork, create a copy of the application template ([`applications/application-template.md`](./applications/application-template.md)). Make sure you create a new file and copy the [contents](https://raw.githubusercontent.com/w3f/Open-Grants-Program/master/applications/application-template.md) of the template inside the new one, and _do not modify the template file directly._
```

Reply:
It is true that using the web interface you can't just copy the file.